actors/runtime/runtime.go

```diff
@@ -33,8 +33,8 @@ type Runtime interface {
 	// Look up the code ID at an actor address.
 	GetActorCodeCID(addr addr.Address) (ret cid.Cid, ok bool)
 
-	// Randomness returns a (pseudo)random string for the given epoch and tag.
-	GetRandomness(epoch abi.ChainEpoch) abi.RandomnessSeed
+	// Randomness returns a (pseudo)random byte array from the chain at a given epoch
+	DrawRandomnessFromChain(epoch abi.ChainEpoch) []byte
```

Comment:
As discussed offline, it makes sense from my perspective for runtime the runtime endpoint to contain all the necessary context for creating the randomness (i.e. DrawRandomness with miner addr and epoch, DrawRandomness with just epoch, DrawRandomness).  Then all the utilities for creating that randomness can sit outside within the node not the vm code.

Doesn't need to happen in this PR.

Reply:
If you agree with the above it would make sense to specify how to do randomness drawing in the needed cases only in the text part of the spec, removing the randomness utility here.

Not sure if that blocks existing intermediate assumptions /testing capabilities in sp-act (i'm new here).

Reply:
I have previously expressed the opposite point of view, which is that the runtime should be the thinnest interface we can define, and pulling code that can be defined in terms of those primitives into the actor spec is a desirable outcome. For one, it's a lot easier to test the actors than the VMs (and will remain true when there are many impls of each).